knowledge-base/kb-ribbonview-customize-tab-bottom-border-fluent-theme.md

```diff
@@ -24,18 +24,18 @@ res_type: kb
 
 ## Description
 
-By default the selected RadRibbonTab control has a 4 pixels high blue-colored border. The following example shows how to customize this without extracting the ControlTemplate of RadRibbonTab.
+How to change the color and thickness of the blue underline border of the selected tab in RadRibbonView.
 
 ## Solution
 
 To customize the RadRibbonTab in code follow the next few steps:
 
 1. Subscribe to the Loaded event of RadRibbonTab.
-2. Call the [ChildrenOfType<T>()]({%slug common-visual-tree-helpers%}) method with the loaded RadRibbonTab control to get all Border elements. 
+2. In the event handler call the [ChildrenOfType<T>()]({%slug common-visual-tree-helpers%}) method to get all Border elements of the loaded RadRibbonTab. 
 3. Call the FirstOrDefault() Linq method to find the Border with x:Name="SelectedVisual". This is the element presenting the border around the selected tab.
```

Comment:
Probably people should know this, but we can specify a bit. For example, Call the FirstOrDefault method from the System.Linq namespace ..